shared/actions/chat/attachment.js

```diff
@@ -182,62 +156,46 @@ function * onSelectAttachment ({payload: {input}}: Constants.SelectAttachment):
   const channelMap = ((yield call(ChatTypes.localPostFileAttachmentLocalRpcChannelMap, channelConfig, {param})): any)
 
   const uploadStart = yield Saga.takeFromChannelMap(channelMap, 'chat.1.chatUi.chatAttachmentUploadStart')
+  const messageID = uploadStart.params.placeholderMsgID
+
+  const updateLocalMessageTask = yield fork(function * () {
```

Comment:
Are we worried about error cases here. I'm a little hesitant about this pattern. What happens if the receivedMessage never comes (network problem?) then we have these dangling generators that are largely invisible? Should we have some kind of race w/ timeout w/ an error or something? We can move forward with it but might lead to some surprising behavior.